.travis.yml

```diff
@@ -1,7 +1,13 @@
 language: node_js
 node_js:
-  - "0.11"
```

Comment:
Is there a reason we wont test against 0.11?  I'm not sure if it matters, but I know other waterline repos test against 0.11.